src/App.jsx

```diff
@@ -6,8 +6,6 @@ import ProtectedRoute from './components/ProtectedRoute';
 import LandingPage from './components/LandingPage';
 import LoginForm from "./components/LoginForm";
 
-// inside <Routes>
-<Route path="/login" element={<LoginForm />} />
```

Comment:
This was embarrassing.  Just a rogue route.  Cleaned that up for this pull request.